tools/docs/manager.ts

```diff
@@ -73,6 +161,29 @@ sidebar_label: ${displayName}
     }
     md += managerReadmeContent + '\n\n';
 
+    const [featureList] = stringifyIssues(managerIssuesMap[manager]?.features);
+    if (featureList) {
+      md += '## Open feature requests\n\n';
+      md += featureList;
+      md += '\n';
+    }
+
+    const [bugList] = stringifyIssues(managerIssuesMap[manager]?.bugs);
+    if (bugList) {
+      md += '## Open bug reports\n\n';
+      md += bugList;
+      md += '\n';
+    }
+
+    if (featureList || bugList) {
+      const now = DateTime.utc().toFormat('MMMM dd, yyyy');
+      const lists = `list of ${featureList ? 'features' : ''}${
+        featureList && bugList ? ' and ' : ''
+      }${bugList ? 'bugs' : ''}`;
+      md += '\n\n';
+      md += `The above ${lists} were current at time of page generation on ${now}.\n`;
```

Comment:
```suggestion
      md += `The above ${lists} were current when this page was generated on ${now}.\n`;
```

I like this wording better, what do you guys think?